### ISSUE TITLE
visionfive2-pvr-graphics: failing installation with SysV init

### ISSUE THREAD

**Issue by OldManYellsAtCloud** [riscv/meta-riscv#473]
**Description**

When `systemd` is not in `DISTRO_FEATURES`, visionfive2-pvr-graphics fails to install the corresponding init scripts. Due to an incorrect if condition, the init system check is always evaluated as systemd.

In line 35 `if [ ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)} ]; then` - though the string returned is correct, the `[]` test always evaluates it as a `true` value, as it checks only string length.

Basically this will always evaluate as true, and echoes:
```
if [ false ]; then
  echo false
fi
```

The correct version would be
```
if false; then
  echo false
fi
```
which evaluates as it is intended to be.
